main.js

```diff
@@ -0,0 +1,28 @@
+const header = document.querySelector("li");
+const readTime = document.querySelectorAll("span");
+const articles = document.querySelectorAll("article");
+
+// Removes only the first li, admittedly.
+if (header.textContent.includes("All Posts")) header.remove();
+
+// iterate over node list to remove spans that include string "Read Time".
+readTime.forEach((span) => {
+  if (span.textContent.includes("Read Time")) {
+    span.remove();
+  }
+});
+
+//uh.
+articles.forEach((article, i) => {
+  if (articles.length - 1 === i) {
+    article.remove();
+  }
+  i++;
+});
+
+// got random.
+// accumulator that iterates over all header type and removes each one systematically.
+for (let i = 1; i < 7; i++) {
+  const currentHeader = document.querySelectorAll(`h${i}`);
```

Comment:
This could also be done by doing `document.querySelectorAll(`h1, h2, h3, ...`).forEach(el => el.remove())`, though the problem here just wants you to remove the h3's in the posts.